src/Operations.jl

```diff
@@ -666,5 +667,22 @@ function test(env::EnvCache, pkgs::Vector{PackageSpec}; coverage=false)
                  " errored during testing")
     end
 end
+
+function init(path::String)
+    gitpath = nothing
+    try
+        gitpath = git_discover(path, ceiling = homedir())
+    catch err
+        err isa LibGit2.GitError && err.code == LibGit2.Error.ENOTFOUND || rethrow(err)
+    end
+    path = gitpath == nothing ? path : dirname(dirname(gitpath))
+    try mkpath(path) end
```

Comment:
Seems better to us `ispath(path) || mkpath(path)`. I know this isn't performance sensitive code but avoiding try/catch wherever possible is still better.